hikari/config.py

```diff
@@ -397,8 +397,72 @@ class that will **NOT** enforce SSL verification. This is then stored
 class CacheSettings:
     """Settings to control the cache."""
 
```

Comment:
also if this is a lib-wise config then it might be worth it having a property on the cache which exposes it's settings

Reply:
Good idea, will add